### ISSUE TITLE
fix tests not working on training set for sides

### ISSUE THREAD

**Issue by jskye** [jskye/haar-classifier-tute#24]
this was because most of the time the bounding box (of the car) was outside the image size.
to fix this, we can just "take a step back" to "get a clearer picture", by embedding the image inside a bigger one such that the bounding box around the car can now be within the image.

**Comment by jskye** [comment]
this also however requires a relabelling of the groundtruths, because theyre now at different positions, as the image is pasted into the middle of the image.
eg. `dims: 100x40` -> `dims: 110x110`
labels: `car at: [0,0,90,45]` -> `car at:  [0+(110-100)/2, 0+(110-40)/2, 90, 45]`

**Comment by jskye** [comment]
fixed on commit `7cdaf45`
